packages/datatrak-web/src/api/queries/useTask.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+import { useQuery } from 'react-query';
+import { DatatrakWebTasksRequest } from '@tupaia/types';
+import { get } from '../api';
+
+export const useTask = (taskId?: string) => {
+  return useQuery(
+    ['task', taskId],
```

Comment:
```suggestion
    ['tasks', taskId],
```
I would have a slight preference for using tasks in the query key since it matches the endpoint name and so that it is possible to clear the cache for all the tasks easily using that key